edn/tests/tests.rs

```diff
@@ -53,6 +53,7 @@ fn test_nil() {
 
 #[test]
 fn test_nan() {
+    assert_eq!(nan("#fNaN").unwrap(), Float(OrderedFloat(f64::NAN)));
```

Comment:
good point. In that case, we should explicitly force the /required/ white space.